Tools/AP_Periph/rangefinder.cpp

```diff
@@ -8,90 +8,99 @@
 
 #include <dronecan_msgs.h>
 
-#ifndef AP_PERIPH_PROBE_CONTINUOUS
-#define AP_PERIPH_PROBE_CONTINUOUS 0
-#endif
-
 /*
   update CAN rangefinder
  */
 void AP_Periph_FW::can_rangefinder_update(void)
 {
-    if (rangefinder.get_type(0) == RangeFinder::Type::NONE) {
-        return;
-    }
-#if AP_PERIPH_PROBE_CONTINUOUS
-    if (rangefinder.num_sensors() == 0) {
-        uint32_t now = AP_HAL::millis();
-        static uint32_t last_probe_ms;
-        if (now - last_probe_ms >= 1000) {
-            last_probe_ms = now;
-            rangefinder.init(ROTATION_NONE);
-        }
-    }
```

Comment:
Note: I had to remove this as rangefinder can only be init once so this probe continuous method does not work with multiple rangefinders.